lib/mayaUsd/ufe/ProxyShapeSceneSegmentHandler.cpp

```diff
@@ -54,23 +54,57 @@ Ufe::Selection ProxyShapeSceneSegmentHandler::findGatewayItems_(const Ufe::Path&
 
     // Find the MayaUSD proxyShapes
     for (const auto& stage : getAllStages()) {
-        Ufe::Path proxyShapePath = stagePath(stage);
-        // recall that findGatewayItems searches for descendents of path that are
+        const Ufe::Path proxyShapePath = stagePath(stage);
+        // recall that findGatewayItems searches for descendants of path that are
```

Comment:
Original spelling, with an "e", was correct.

Reply:
Both are correct, but according to Merriam-Webster spelling with an 'a' is more common:
https://www.merriam-webster.com/dictionary/descendant

Reply:
I learned something today. Thanks!

Reply:
this is why I love code reviews... and just a heads up, I am terrible at the English language :-)